composer.json

```diff
@@ -46,14 +46,14 @@
         ],
         "post-install-cmd": [
             "php artisan clear-compiled",
-            "php artisan optimize",
-            "bash bin/setup.sh"
+            "php artisan optimize"
         ],
         "pre-update-cmd": [
             "php artisan clear-compiled"
         ],
         "post-update-cmd": [
-            "php artisan optimize"
+            "php artisan optimize",
+            "bash bin/setup.sh"
```

Comment:
actually needed to sync about this one with you. i understand the part of what happens in post-update-cmd and post-install-cmd, but not sure where to put setup.sh 
tried to get after the composer install/update to run phpunit in vagrant, but it runs in local terminal instead of vagrant... complete mush in my head about this one